docs/experiment-analysis/data-pipeline.md

```diff
@@ -44,4 +44,17 @@ Funnel metrics require their own pipeline because of the complicated logic that
 
 1. **Incremental funnel metric events**: We incrementally join assignment data to the event sources for your funnel metrics.
 2. **Funnel metrics data frame**: We generate a data frame for each subject and day for each funnel metric.
-3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
\ No newline at end of file
+3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
+
+## Privacy
+
+As we’ve detailed, Eppo doesn’t export individual data from your warehouse. We actively minimize what information we store.
+
+* **Fact creation**: We query small sample of tables to help you set-up and make sure the column names and types match. We do not send those results beyond your analysts’ browser nor do we retain them; we just store the configuration. We can anonymize that step by only showing the data type, not the exact value. We do not need access, and encourage you *not* to give our service account access to any personally identifiable information (PII), just unique identifiers that you have generated.
+* **Incremental pipelines**: Our pipelines query daily aggregate of assignments and facts, we only store those on your warehouse. We only keep a copy of the text of the queries that we generate and run (where table and column names appear, but no individual information). We do not keep individual outputs for any daily query. We add them up into incremental tables and store those in your warehouse too.
+* **Auditability**: Our queries and the resulting tables are fully auditable by your data engineers. Our process rely on a service account whose access you control and can revoke at any time. Your engineers send the token for that service account directly to our secure enclave with no intermediaries. You can update that token on your own if needed.
```

Comment:
It's not necessarily just a token.  It's username / password.  So I'd reframe this a bit.

Reply:
Yeah, switched to Credentials.
Thanks for the review!